book_maker/cli.py

```
@@ -443,8 +579,23 @@ def main():
     )
+    # Parse and set extra_body if provided
+    if options.extra_body:
+        try:
+            import json
+
+            extra_body = json.loads(options.extra_body)
+            e.translate_model.extra_body = extra_body
+            print(f"[bold blue]Extra body parameters:[/bold blue] {extra_body}")
+        except json.JSONDecodeError as e:
+            print(f"[bold red]Error:[/bold red] Invalid JSON in --extra_body: {e}")
+            exit(1)
```

Ruff (0.15.12)

[warning] Use `sys.exit()` instead of `exit`

Replace `exit` with `sys.exit()`

(PLR1722)